scipy/sparse/linalg/dsolve/SuperLU/License.txt

```diff
@@ -27,3 +27,40 @@ PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
 LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
 NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. 
+
+
+-----------------
+External software
+-----------------
+In the ILU preconditioner driver routine xGSISX, we use MC64 routine
+to pre-permute large entries to the main diagonal. This use path can
+be removed for commercial users. MC64 routine carries the following notice:
```

Comment:
The MC64 file has not been added to Scipy (due to incompatible license), so to avoid confusion this section should be removed from the license file.

Reply:
I removed this section (which resulted in an identical file to what's checked in).
This PR was mostly a bulk copy and I didn't process the implications of changing this file, I appreciate that you read this change and understood it. Thanks.